test/mongorito.test.js

```diff
@@ -1120,7 +1109,45 @@ describe ('Mongorito', function () {
     secondaryDb.close();
   });
 
+  it ('be able to yield a close call (issue #23)', function * (){
+    let secondaryDb = yield Mongorito.connect('localhost/mongorito_test_2');
+    yield secondaryDb.close();
```

Comment:
If `close()` is yieldable, it should probably be tested to actually close the connection